wallets/provider-phantom/src/mod.ts

```diff
@@ -0,0 +1,11 @@
+import { defineVersions } from '@rango-dev/wallets-core/utils';
+
+import { legacyProvider } from './legacy/index.js';
+import { provider } from './provider.js';
+
+const versions = defineVersions()
```

Comment:
As a suggestion, can't we handle this by exporting `legacy` and `hub` separately instead of this `versions` approach? Maybe that way we can prevent including `legacy` or `hub` code based on the value of `experimentalWallet` in `config`.

Reply:
We've talked about this offline, I mentioned about parcel has struggling with `exports` filed and also versioning maybe removed in future versions since it's kinda experimental.

So I prefer to keep it this way yet.